deploy-service/universal/src/test/java/com/pinterest/teletraan/universal/events/GenericEventPublisherTest.java

```diff
@@ -9,11 +24,10 @@
 import static org.mockito.Mockito.when;
 
 import java.util.concurrent.TimeUnit;
-
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-public class GenericEventPublisherTest {
+class GenericEventPublisherTest {
```

Comment:
why remove? better to make it explicit?

Reply:
It doesn't need to be public.

Reply:
https://rules.sonarsource.com/java/RSPEC-5810/